pom.xml

```diff
@@ -113,6 +114,11 @@
             <artifactId>pipeline-stage-step</artifactId>
             <version>2.2</version>
         </dependency>
+        <dependency>
+            <groupId>org.jenkinsci.plugins</groupId>
+            <artifactId>pipeline-stage-tags-metadata</artifactId>
```

Comment:
This was needed to avoid hard-coding some strings related to the stage tag, which is used to determine if the stage was skipped for conditional.

Reply:
Technically allowable but I'd *really* rather we just hardcode the string because this carries along a *giant* wad of dependencies.

Reply:
Actually, that one has no dependencies. Deliberately. =)

Reply:
Oh you're right, I was looking at the root pom.   Still I don't think it's worth taking a dependency just to carry a single string.

Reply:
There are technically _two_ string values we're pulling in here via `StageStatus.getSkippedForConditional()` (`"SKIPPED_FOR_CONDITIONAL"`) and `StageStatus.TAG_NAME` (`"STAGE_STATUS"`). Not sure if that changes it at all for you, guessing probably not... I can just encapsulate those values in this plugin's own `StageStatus` class and add a comment referencing the other plugin.

Reply:
I removed this dependency in commit 40dc7ba